include/pmacc/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ get_filename_component(PMACC_ROOT_DIR "${PMACC_ROOT_DIR}" ABSOLUTE)
 # PMacc
 ################################################################################
 
-find_package(PMacc REQUIRED CONFIG PATHS ${CMAKE_CURRENT_SOURCE_DIR})
+find_package(PMacc REQUIRED CONFIG PATHS test)
```

Comment:
are `${CMAKE_CURRENT_LIST_DIR} == ${CMAKE_CURRENT_SOURCE_DIR}) == test`?

Reply:
no left over from a test, good catch

Reply:
Is it still "test"? (At least I seee "test" in the code).